harvester_e2e_tests/integration/test_images.py

```diff
@@ -0,0 +1,375 @@
+
+    while endtime > datetime.now():
+        code, data = api_client.images.get(display_name)
+        image_status = data.get('status', {})
+
+        assert 200 == code, (code, data)
+        if image_status.get('progress') == 100:
+            break
+        sleep(5)
+    else:
+        raise AssertionError(
+            f"Failed to download image {display_name} with {wait_timeout} timed out\n"
+            f"Still got {code} with {data}"
+        )
+
+
+def delete_image(api_client, unique_name, wait_timeout):
+    code, data = api_client.images.delete(unique_name)
+
+    assert 200 == code, (f"Failed to delete image with error: {code}, {data}")
+
+    endtime = datetime.now() + timedelta(seconds=wait_timeout)
+
+    while endtime > datetime.now():
+        code, data = api_client.images.get(unique_name)
+        if code == 404:
+            break
+        sleep(5)
+    else:
+        raise AssertionError(
+            f"Failed to delete image {unique_name} with {wait_timeout} timed out\n"
+            f"Still got {code} with {data}"
+        )
+
+
+def delete_volume(api_client, unique_name, wait_timeout):
+    code, data = api_client.volumes.delete(unique_name)
+
+    assert 200 == code, (f"Failed to delete volume with error: {code}, {data}")
+
+    endtime = datetime.now() + timedelta(seconds=wait_timeout)
+    while endtime > datetime.now():
+        code, data = api_client.volumes.get(unique_name)
+        if code == 404:
+            break
+        sleep(5)
+    else:
+        raise AssertionError(
+            f"Failed to delete volume {unique_name} with {wait_timeout} timed out\n"
+            f"Still got {code} with {data}"
+        )
+
+
+def get_volume(api_client, unique_name):
+    code, data = api_client.volumes.get(unique_name)
+
+    assert 200 == code, (code, data)
+    assert unique_name == data['metadata']['name']
+
+
+def get_image(api_client, unique_name):
+    code, data = api_client.images.get()
+
+    assert len(data['items']) > 0, (code, data)
+
+    code, data = api_client.images.get(unique_name)
+    assert 200 == code, (code, data)
+    assert unique_name == data['metadata']['name']
+
+
+def create_with_invalid_url(api_client, unique_name):
+    code, data = api_client.images.create_by_url(unique_name, f"https://{unique_name}.img")
+
+    assert 201 == code, (code, data)
+
+    endtime = datetime.now() + timedelta(minutes=3)
+    while endtime > datetime.now():
+        code, data = api_client.images.get(unique_name)
+        image_conds = data.get('status', {}).get('conditions', [])
+        if len(image_conds) > 0:
+            break
+        sleep(3)
+
+    assert len(image_conds) == 1, f"Got unexpected image conditions!\n{data}"
+    assert "Initialized" == image_conds[0].get("type")
+    assert "False" == image_conds[0].get("status")
+    assert "no such host" in image_conds[0].get("message")
+
+    api_client.images.delete(unique_name)
+
+
+class TestBackendImages:
+
+    @pytest.mark.images_p1
```

Comment:
`images_p1` is deprecated markers, update them to use `images` and `p0/p1/p2`, to notice the highest priority is `p0`.

and can just mark the class, it will mark all methods inside.

Reply:
Thanks for the review. 
Mark test class and test cases with `pytest.mark.p0` fixture